### DISCUSSION TITLE
Source code for https://nextjs.org/blog page

### DISCUSSION THREAD

**Discussion by sibasishm** [vercel/next.js#14914] [Help] [answered]
I am curious how the blog page in next.js official site is made and would be great if the team can provide a link to the source code or anything similar to that. Thank you.

**Comment by c0b41** [comment, marked as answer]
https://github.com/vercel/next-site

**Reply by ldalves** [reply]
`404`

Does it have a new repo?

**Reply by theapplegates** [reply]
Curious about this myself. Any updated answers?

**Reply by ghost** [reply]
I got in contact with Vercel and the blog source code is now private.
